File: PST/FUEL Source Code/FUEL/TestComplete.cs
// Decompiled with JetBrains decompiler
// Type: FUEL.TestComplete
// Assembly: FUEL, Version=2.4.8.0, Culture=neutral, PublicKeyToken=null
// MVID: A3D630D0-0620-425D-8CFB-B67B2AC75507
// Assembly location: C:\hp\FUEL\FUEL.dll

using Microsoft.VisualBasic.CompilerServices;
using System.Diagnostics;
using System.Runtime.CompilerServices;
using System.Windows.Forms;

namespace FUEL
{
  public class TestComplete : FSDialog
  {
    private object _Passed
    {
      [DebuggerNonUserCode] get => this.__Passed;
      [DebuggerNonUserCode] set => this.__Passed = RuntimeHelpers.GetObjectValue(value);
    }

    public TestComplete(bool Passed) => this._Passed = (object) Passed;

    public void Show() => this.Show((Form) new dlgTestComplete(Conversions.ToBoolean(this._Passed)));
  }
}

File: PST/FUEL/src/FUEL/TestComplete.vb
Imports Microsoft.VisualBasic.CompilerServices
Imports System
Imports System.Diagnostics
Imports System.Runtime.CompilerServices

Namespace FUEL
    Public Class TestComplete
        Inherits FSDialog
        ' Methods
        Public Sub New(ByVal Passed As Boolean)
            Me._Passed = Passed
        End Sub

        Public Sub Show()
            Dim dialog As New dlgTestComplete(Conversions.ToBoolean(Me._Passed))
            Me.Show(dialog)
        End Sub


        ' Properties
        Private Property _Passed As Object
            <DebuggerNonUserCode> _
            Get
                Return Me.__Passed
            End Get
            <DebuggerNonUserCode> _
            Set(ByVal AutoPropertyValue As Object)
                Me.__Passed = AutoPropertyValue
            End Set
        End Property


        ' Fields
        <CompilerGenerated, DebuggerBrowsable(DebuggerBrowsableState.Never)> _
        Private __Passed As Object
    End Class
End Namespace

